src/scss/block/link.scss

```diff
@@ -21,6 +21,11 @@
         .deleted {
             .icon.ghost { width: 24px; height: 24px; }
         }
+		
+		.emptyObjectLink{
+    		border-bottom: $colorDarkGrey 0.05em dashed;
+    		line-height: 22px;
+		}
 
```

Comment:
Can't user empty, it's already defined in common.scss

Reply:
It doesnt matter, we have a design system and this is out of design system that we have, we do not use dashed borders anywhere, color of the borders should be one of colorShape<...> and formatting is still not fixed.

Reply:
I'm not sure what to say; I might have missed something.  
I tried to use what was already in the code (colorDarkGrey and then colorTextSecondary... now colorShapePrimary, although I'm not convinced about visibility with this "almost white"). I also asked on the forum and on Slack because, yes, I understand that design is your responsibility, and the appearance of "empty link" doesn't exist yet. Or should I really use the existing .empty? (I looked into it, and it didn't seem relevant, but why not).
Or perhaps this design system can be grasped by participants, but I missed it (Anytype documentation, recognized standards, etc.). Sorry for that.
Perhaps it would be best to simply specify not to touch the code when it involves CSS?

Reply:
`.emptyObjectLink { border-bottom: $colorShapePrimary 0.05em dashed; line-height: 22px; }`

Reply:
Hello @Shampra, 

I'm Alex from Anytype's design team.

Thank you for your input. We are seeing link styles as a part of our design system, so we re avoiding making isolated changes. We'll keep your suggestion in mind and revisit it when we review link styles.

Cheers,
Alex

Reply:
Hello @sebyakin ,
Thank you for your feedback. It's important for me to know, especially since I was in the midst of  of a more significant task regarding links to address several requests (integrating more elements from the linked block). I'll stop here for now ;-).

I'll modify as requested by @ra3orblade (thank you too) and  I'll leave it up to you to determine if this contribution is acceptable or should be set aside.